internal/core/ngt/option.go

```diff
@@ -94,16 +94,18 @@ func WithDistanceTypeByString(dt string) Option {
 		d = L1
 	case "l2":
 		d = L2
-	case "angle":
+	case "angle", "ang":
 		d = Angle
-	case "hamming":
+	case "hamming", "ham":
 		d = Hamming
 	case "cosine", "cos":
 		d = Cosine
-	case "normalizedangle":
+	case "normalizedangle", "nang", "nangle":
 		d = NormalizedAngle
-	case "normalizedcosine":
+	case "normalizedcosine", "nham", "nhamming":
```

Comment:
i don't know but "normalized cosine" is same as "normalized hamming"?

Reply:
sorry it's my mistake, when I open this PR I was drunk :beer:

Reply:
@rinx sorry I fixed you pointed out.